katello_certs/hooks/boot/02-message-helpers.rb

```diff
@@ -62,8 +62,7 @@ def proxy_instructions_message(kafo)
                     --foreman-proxy-trusted-hosts                 "#{fqdn}"\\
                     --foreman-proxy-trusted-hosts                 "#{foreman_proxy_fqdn}"\\
                     --foreman-proxy-oauth-consumer-key            "#{foreman_oauth_key}"\\
-                    --foreman-proxy-oauth-consumer-secret         "#{foreman_oauth_secret}"\\
-                    --puppet-server-foreman-url                   "#{foreman_url}"
```

Comment:
Removing this does break it for Puppet. Should we keep a note for those users similar to what we've done in the hostname rename tool?